CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(roboteq_driver)
+
+set(CMAKE_CXX_STANDARD 14)
+
+include_directories(include)
+
+add_library(${PROJECT_NAME}
+            src/rawcan_comm.cpp
+            src/roboteq_controller.cpp)
+
```

Comment:
`target_compile_options(${PROJECT_NAME} -Wall -Wextra -Wpedantic -Werror)`

These compiler options will help prevent undeterministic behaviour. you'll have to fix them before it'll build

Reply:
`CMake Error at CMakeLists.txt:14 (target_compile_options):
  target_compile_options called with invalid arguments


-- Configuring incomplete, errors occurred!
See also "/home/akeaveny/cpp/roboteq_driver/CMakeFiles/CMakeOutput.log".
Makefile:236: recipe for target 'cmake_check_build_system' failed
make: *** [cmake_check_build_system] Error 1
`

Reply:
I keep getting this error